### PR TITLE
Magnite Analytics Adapter : track to PBS Analytics Event and ATAG

### PR DESCRIPTION
**Pull request by spotxslagle**
## Type of change

- [ x ] Feature

## Description of change

Subscribes the MAA to the PBS Analytics event and captures Analytics tag data

## Other information

Supercedes https://github.com/prebid/Prebid.js/pull/12000

### CODE REVIEW

File: modules/magniteAnalyticsAdapter.js [hunk 1]
```diff
@@ -927,8 +927,8 @@ magniteAdapter.track = ({ eventType, args }) => {
       const bidStatus = args.rejectionReason === REJECTION_REASON.FLOOR_NOT_MET ? BID_REJECTED_IPF : 'rejected';
       handleBidResponse(args, bidStatus);
       break;
-    case SEAT_NON_BID:
-      handleNonBidEvent(args);
+    case PBS_ANALYTICS:
```
Comment (patmmccann) [inline modules/magniteAnalyticsAdapter.js:930]: Since you stop listening to non bid, is this blocked by #12044 ?

**Comment by patmmccann** [comment]
Appears to be blocked by #12044

File: modules/magniteAnalyticsAdapter.js (hunk 1 above)
Comment (robertrmartinez) [reply modules/magniteAnalyticsAdapter.js:930]: yeah it is

**Review by robertrmartinez — APPROVED** [review]

**Comment by spotxslagle** [comment]
@patmmccann Other change got merged, so this is no longer blocked

**Comment by robertrmartinez** [comment]
Hey all can we get a second non-magnite reviewer please?

@patmmccann @ChrisHuie

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**